app/assets/stylesheets/govuk_publishing_components/components/_contents-list.scss

```diff
@@ -76,6 +76,27 @@
   }
 }
 
+.gem-c-contents-list--alternate-active {
+  .gem-c-contents-list__list-item {
+    padding: govuk-spacing(2) 0;
+    margin: 0;
+
+    & span::before {
```

Comment:
I know you didn't add this span, but can we take the opportunity to give it a proper class? Something like `.gem-c-contents-list__dash` or something.

Reply:
Since that applies to many other variants, should that be in this PR?

Reply:
I think that's fine and it should be a small change - just add the class in the markup and adjust the instances of `span` in the Sass to the new classname.

Reply:
OK this is now done.